src/server/pwa/middleware/service-worker.js

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import fs from 'fs';
+import path from 'path';
+
+let serviceWorkerEnabled = false;
+let serviceWorkerScript = null;
+let serviceWorkerScope = null;
```

Comment:
Yes, I think it is needed here to avoid circular dependencies between importing the files since `pwa/config.js` is importing `configureServiceWorker` from `pwa/middleware/service-worker.js`.

Reply:
We'll need a solution that allows us to track this  value in one place

Reply:
Got it I will update.

Reply:
I've updated to place all config in `server/middleware/pwa/config.js` (fe13f69)